target/linux/mvebu/base-files/etc/board.d/01_leds

```diff
@@ -41,6 +41,12 @@ armada-385-linksys-venom)
 	ucidef_set_led_usbport "usb2" "USB 2" "pca963x:venom:blue:usb3_1" "usb2-port1" "usb3-port1"
 	ucidef_set_led_usbport "usb2_ss" "USB 2 SS" "pca963x:venom:blue:usb3_2" "usb3-port1"
 	;;
+armada-385-wd-mcex2u)
+	# ucidef_set_led_default "hdd1" "HDD1 Activity" "wdmcex2u:blue:hdd1" "1"
```

Comment:
If you don't need those, then remove it, but don't keep this commented out code.